vuu-ui/packages/vuu-table/src/table-header/useTableHeader.ts

```diff
@@ -14,13 +14,38 @@ export interface TableHeaderHookProps
   onSortColumn: (column: ColumnDescriptor, addToExistingSort: boolean) => void;
 }
 
+// const mouseEnterHandler = (e: MouseEvent) => {
+//   console.log("mouse enter", {
+//     target: e.target,
+//   });
+// };
+// const mouseLeaveHandler = (e: MouseEvent) => {
+//   const el = e.target as HTMLElement;
+// };
+
```

Comment:
Should we remove this commented out code if not needed?

And the one below this as well.